tuw_nlp/graph/utils.py

```diff
@@ -152,8 +165,9 @@ def max_distance(self, graph, node_data, subgraphs):
         if not possible:
             return False
         for n1, n2 in product(node1_matches, node2_matches):
-            n1_root = [n for n, d in n1.in_degree() if d == 0][0]
-            n2_root = [n for n, d in n2.in_degree() if d == 0][0]
+            # This is a temporary fix. Should be d==0 for both.
+            n1_root = [n for n, d in n1.in_degree() if d < 2][0]
```

Comment:
I have changed this as a workaround solution for using the path() function with entities. Particularly, if I want to look for a path to or from any arbitrary entity 1, the penman code looks like this: ( .* / u-1 :entity (e_1 / 1 )). This seems to create a single node subgraph with one edge. The list returned by this comprehension is therefore empty, and the reference to element 0 will throw an exception. The same workaround is included in the path_between function